tests/libcxx/expected_results.txt

```diff
@@ -530,6 +530,11 @@ std/thread/macro.pass.cpp:1 FAIL
 std/depr/depr.c.headers/tgmath_h.pass.cpp:1 FAIL
 
 
+# *** CLANG FEATURES NOT YET IMPLEMENTED ***
+# P0960R3 "Allow initializing aggregates from a parenthesized list of values"
+std/utilities/tuple/tuple.tuple/tuple.creation/tuple_cat.pass.cpp SKIP # TRANSITION, VS 2019 16.8 p3

```

Comment:
FYI, I chose to mark this `SKIP` since we can't currently fix-and-update-from-upstream due to the breakage witnessed in #1090. The TRANSITION comment is here to remind us to change this to `FAIL:0` after the MSVC with the new feature is released if we still haven't managed to fix this upstream.

Reply:
I also apparently chose `SKIP` because I had forgotten the correct status is `SKIPPED`:
```suggestion
std/utilities/tuple/tuple.tuple/tuple.creation/tuple_cat.pass.cpp SKIPPED # TRANSITION, VS 2019 16.8 p3
```